docs/programming/kotlin/fundamentals/collections.md

```diff
@@ -0,0 +1,32 @@
+# Collections
+
+## Collection & Iterable Interface
+
+*Collection <T>* is the root of the collection hierarchy. The collection interface extends the Iterable interface. The 
+`Iterable interface` defines the operations for iterating over elements. For example, elements of an array. Since the 
+Collection interface inherits from the Iterable Interface, it also inherited helpful methods that can help us iterate or loop 
+over elements. The Collection interface also has its own helpful implementations of methods for its subclasses as well. 
+Such as getting the size, checking item membership, checking if two collections are equal, and so on.
+
+***This diagram shows the hierarchy:***
+![Collections Diagram](https://cdn.discordapp.com/attachments/956755409878786088/1138236808426176562/collections-diagram.png)
+
```

Comment:
Would talk about collection methods first. Would then allow you to reduce overlap between lists, sets, and maps (as they all extend Collection)